users/templates/emails/create_user_email.html

```diff
@@ -0,0 +1,10 @@
+{% autoescape off %}
+<p> Hi {{ user_name }}! </p>
+
+<p> Your Sheetstorm account has been created. You can login at http://{{ domain }}. Your temporary password is: <b>{{ password }}</b> </p>
```

Comment:
Okay, I see the problem with `http` now. I created separate task for this https://trello.com/c/mfvSVFx1/44-add-siteurl-setting-to-distinguish-proper-address. So just leave it here as it is.

Reply:
Ok. In CustomPasswordChangeView I use request.scheme to do that.